### PR TITLE
Refactor session cache in MetricsPersistentWorker.

### PR DESCRIPTION
**Pull request by wu-sheng**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

This is another preparation for #10051. After this, I would be more confident to move the booting timestamp as a time point to control whether metrics should be loaded from the database or return not existing if it can't be found in the cache.
Also, this makes the cache controller out of the worker process.

### CODE REVIEW

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsSessionCache.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import lombok.Setter;
+import org.apache.skywalking.oap.server.core.analysis.metrics.Metrics;
+import org.apache.skywalking.oap.server.library.client.request.InsertRequest;
+import org.apache.skywalking.oap.server.library.client.request.UpdateRequest;
+
+/**
+ * MetricsSessionCache is a key-value cache to hold hot metric in-memory to reduce payload to pre-read.
+ *
+ * There are two ways to make sure metrics in-cache,
+ * 1. Metrics is read from the Database through {@link MetricsPersistentWorker}.loadFromStorage
```
Comment (kezhenxu94) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsSessionCache.java:34]: 👇 suggestion so we can directly jump to the method?

```suggestion
 * 1. Metrics is read from the Database through {@link MetricsPersistentWorker#loadFromStorage}
```

**Review by wankai123 — APPROVED** [review]

**Review by wankai123 — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]
One nit

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsSessionCache.java (hunk 1 above)
Comment (wu-sheng) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsSessionCache.java:34]: Tried and failed :P This is a private method in the worker

image

Comment (kezhenxu94) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsSessionCache.java:34]: 🥹 ok then. LGTM

**Pull request merged**